doc/languages.rst

```diff
@@ -389,6 +390,11 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check syntax and types of Go tests with the `Go compiler`_.
 
+      .. option:: flycheck-go-build-install-deps
```

Comment:
Please add a `:noindex:` option to this directive.  You document the same symbol twice, or it'll mess up cross-referencing.